chroma_core/models/host.py

```diff
@@ -1578,22 +1578,13 @@ class Meta:
         app_label = "chroma_core"
 
 
-class WriteConfStep(Step):
+class ReplaceNidsStep(Step):
     def run(self, args):
-        from chroma_core.models.target import FilesystemMember
-
-        target = args["target"]
-
-        agent_args = {"erase_params": True, "device": args["path"]}
-
-        if issubclass(target.downcast_class, FilesystemMember):
-            agent_args["mgsnode"] = args["mgsnode"]
-            agent_args["writeconf"] = True
-
-        fail_nids = args["fail_nids"]
-        if fail_nids:
-            agent_args["failnode"] = fail_nids
-        self.invoke_agent(args["host"], "writeconf_target", agent_args)
+        device = args["path"]
+        nids = args["nids"]
+        flattened_nids = [e for t in nids for e in t]
```

Comment:
Technically, `flattened_nids` and `nids` may include duplicate elements. Is it ok?

Reply:
I think that shouldn't be the case based on what I've seen. But it's not handled here, just as in previous implementation.